### PR TITLE
DCGAN 학습 완료...?

### PR DESCRIPTION
**Pull request by TetorCo**
※ '기능 구현 완료' or '문서화' 경우에 맞는 양식 사용하시면 됩니다.  
※ 각 양식 상단에 안내드린 값에 맞춰 Pull Request 제목과 우측 Label 등 지정해주세요.  
※ 작성 시 **굵게 표시한 것**과 _기울임_ 한 것은 지우셔도 괜찮습니다.

---
_name: feature_made  
about: 기능 구현 완료 후 Pull Request 작성 시 사용할 Template  
title: "[FeatureMade] 제목"  
labels: ["분류: 기능 구현"]  
assignees: 'seahahn'_

**※ 이슈에 명시된 기능 구현 완료한 경우 작성하기**  
**※ 구현한 기능이 명시된 Issue를 꼭! 연결시키기**  
**※ 가능하면 기능 설명을 소스 코드의 해당 부분에 docstring 식으로 붙여두는 것을 추천합니다.**  

### 작성자
김태범

### 1. 기능 설명
DCGAN을 활용한 새로운 이모티콘 생성 모델 만들기, StarGAN Commit

### 2. 인풋(매개변수) & 아웃풋(return 값) (optional)
_구현한 기능이 함수인 경우 인풋과 아웃풋에 대한 설명을 남겨주세요.  
ex. 인풋 : JSON으로 변환된 데이터프레임  
아웃풋 : 함수 연산을 거친 후 JSON으로 변환된 데이터프레임_

### 3. 추가 내용 (optional)
_위 사항들 외 별도로 남기실 내용을 작성해주세요.  
(텍스트 외 스크린샷 등의 이미지 첨부 가능)_

---
_name: documentation  
about: 문서 내용 추가 또는 수정 시 사용할 Pull Request Template  
title: "[DOCU] 제목"  
labels: ["분류: 문서화"]  
assignees: 'seahahn'_

**※ 문서 내용 추가 또는 수정을 한 경우 작성하기**

### 작성자
_여기에_이름을_남겨주세요_

### 1. 문서화 내용
_어떤 내용을 추가 또는 수정하셨는지 간략히 남겨주세요._

### 2. 추가 내용 (optional)
_위 사항들 외 별도로 남기실 내용을 작성해주세요.  
(텍스트 외 스크린샷 등의 이미지 첨부 가능)_

### CODE REVIEW

File: StarGAN/test.py [hunk 1]
```diff
@@ -5,9 +5,10 @@
 import image_proprecessing
 import os
 
-Gen = Generator(64, 5)
-G_path = 'D:/Ai Play/StarGAN/200000-G.ckpt'  # 200000번 학습한 파라미터를 가져옴.
+Gen = Generator(64, 5)  # Generator 생성
+G_path = 'D:/AIPlay/DL-API/StarGAN/200000-G.ckpt'  # 200000번 학습한 파라미터를 가져옴.
```
Comment (seahahn) [inline StarGAN/test.py:9]: gitignore는 안 보이는데요, 파일 크기 때문에 별도로 제외하신 건가요?

File: StarGAN/test.py [hunk 2]
```diff
@@ -27,13 +28,16 @@
 G_dict.update(G_new_pretrained_dict)
 # 미리 학습된 파라미터를 적용
 Gen.load_state_dict(G_new_pretrained_dict)
+##########################################################################
 
-result_dir = 'D:/Ai Play/StarGAN/results'
+result_dir = 'D:/AIPlay/DL-API/StarGAN/results'
```
Comment (seahahn) [inline StarGAN/test.py:33]: 각자 환경이 다르니 상대 경로로 지정하시는 것을 추천드립니다.

File: StarGAN/test.py [hunk 3]
```diff
@@ -27,13 +28,16 @@
...
 if not os.path.exists(result_dir):
     os.makedirs(result_dir)  # 만약 results 파일이 없으면 생성
 
 ## Test ##
-image_dir = 'D:/Ai Play/DL-API/StarGAN'
+# 이미지 같은 경우는 나중에 고민 해야하는 문제
+# 경로가 아니라 이미지 자체를 받을 수도 있음
+image_dir = 'D:/AIPlay/DL-API/StarGAN'
```
Comment (seahahn) [inline StarGAN/test.py:40]: 마찬가지, 상대 경로 지정 추천드려요.

File: StarGAN/test.py (hunk 1 above)
Comment (TetorCo) [reply StarGAN/test.py:9]: 모델을 따로 저장하지 않고 .ckpt 파일을 사용해서 8번 라인에서 생성한 Generator 모델에 미리 20만번 학습한 가중치들을 덮어씌워주는 형식입니다.

File: StarGAN/test.py (hunk 2 above)
Comment (TetorCo) [reply StarGAN/test.py:33]: 네 알겠습니다.

File: StarGAN/test.py (hunk 1 above)
Comment (seahahn) [reply StarGAN/test.py:9]: > 모델을 따로 저장하지 않고 .ckpt 파일을 사용해서 8번 라인에서 생성한 Generator 모델에 미리 20만번 학습한 가중치들을 덮어씌워주는 형식입니다.

그러니까... ckpt 파일이 있는 것이지요? 이 파일이 pull request 목록에 안 보여서 여쭤봤습니다.
서버에 올리려면 파일 크기도 고려를 해야 해서요.
